run-tests.sh

```diff
@@ -20,7 +20,7 @@ function cleanup {
 }
 trap cleanup EXIT
 
-python -m check_manifest --ignore ".*-requirements.txt"
+python -m check_manifest --ignore ".*-requirements.txt,pyproject.toml"
```

Comment:
I am still not convinced about adding this, it "only" affects those installing local modules via pipenv... From my perspective, either we add it everywhere either has to be added locally somehow.

Reply:
Sorry! This was not meant to be committed. I am only doing it locally. Removing!